source/gui/__init__.py

```diff
@@ -559,8 +559,7 @@ class WelcomeDialog(wx.Dialog):
 	This dialog is displayed the first time NVDA is started with a new configuration.
 	"""
 
-	WELCOME_MESSAGE = _(
-		"Welcome to NVDA!\n"
+	WELCOME_MESSAGE_DETAIL = _(

```

Comment:
Can you please add a translators comment here? I realise there wasn't one before. :(